GeeksCoreLibrary/Modules/PostalServices/PostNL/Models/CustomsContentModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace GeeksCoreLibrary.Modules.PostalServices.PostNL.Models
+{
+    public class CustomsContentModel
+    {
+        public string CountryOfOrigin { get; set; }
+        public string Description { get; set; }
+        public string HSTariffNr { get; set; }
```

Comment:
Kun je hier ook de naam aanpassen zodat er geen afkortingen meer in staan en geen meerdere hoofdletters achter elkaar?